test/cmd/basicresources.sh

```diff
@@ -154,7 +146,7 @@ os::cmd::expect_success 'oc get nodes'
   # subshell so we can unset kubeconfig
```

Comment:
can delete this whole subshell stanza

Reply:
> can delete this whole subshell stanza

You don't think the point of the test was to see if an explicit kubeconfig worked?  I thought the `kubectl` aspect was ancillary.

Reply:
yeah, sorry... misread the change and though you removed line 157 completely